lightclient-circuits/src/gadget/crypto/sha256_wide/gate.rs

```diff
@@ -45,6 +42,27 @@ pub struct ShaBitGateManager<F: Field> {
     pub copy_manager: SharedCopyConstraintManager<F>,
 }
 
+/// Witnesses of a sha256 which are necessary to be loaded into halo2-lib.
+#[derive(Clone, Copy, Debug, CopyGetters, Getters)]
+pub struct LoadedSha256<F: Field> {
+    /// bytes_left of the first row of the first round of this keccak_f. This could be used to determine the length of the input.
```

Comment:
Whys this commented out?

Reply:
coppied it from fork, `bytes_left` was used there when for dynamic size message, which we don't use here